### PR TITLE
add more ts mode from tree-sitter-module

### PR DESCRIPTION
**Pull request by getong**
This is a copy from https://github.com/casouri/tree-sitter-module
just copy the url from build.sh

### CODE REVIEW

**Comment by renzmann** [comment]
Thanks for contributing all these!  I'm going to wait on #43, #53, and #54 before diving into these too deep, since it looks like there's some overlap between the modes they would be remapping alongside these.

File: treesit-auto.el [hunk 1]
```diff
@@ -247,11 +267,42 @@ automatic installation (or prompting, based on the value of
       :url "https://github.com/ikatyang/tree-sitter-markdown"
       :ext "\\.md\\'")
     ,(make-treesit-auto-recipe
-      :lang 'proto
```
Comment (renzmann) [inline treesit-auto.el:250]: Issue: It looks like we have an accidental overwrite of the `:lang 'proto` that's required here.

**Review by renzmann — CHANGES_REQUESTED** [review]
One quick thing before we bring these in: mind adding the `:ext ...` keyword to each of these recipes?  (documented [here](https://github.com/renzmann/treesit-auto/blob/main/CONTRIBUTING.org))  This would enable a few features of `treesit-auto` that could enable and/or install these modes based on the file extension.

**Comment by getong** [comment]
surface file is elixir filename extension, ignore it. The other new added lang have added the ext name.

**Pull request closed without merging**